include/oneapi/dpl/pstl/pstl_config.h

```diff
@@ -66,7 +66,8 @@
 #endif
 
 // Enable SIMD for compilers that support OpenMP 4.0
-#if (_OPENMP >= 201307) || (__INTEL_COMPILER >= 1600) || (!defined(__INTEL_COMPILER) && _PSTL_GCC_VERSION >= 40900) || \
+#if (__INTEL_LLVM_COMPILER || __INTEL_COMPILER ||                                                                      \
```

Comment:
it looks that we lost `(_OPENMP >= 201307) ` condition

Reply:
Fixed